Quria/javaScript.js

```diff
@@ -0,0 +1,109 @@
+var popup_modal = document.getElementById("popup-modal");
+var close = document.getElementsByClassName("close")[0];
+
+var books= [
+  {
+     "item_id": "28",
+     "name": "Harry Potter",
+     "picture":"harrypotter.jfif"
+  },
+  {
+     "item_id": "18",
+     "name": "Spegelmannen",
+     "picture":"spegelmannen.jfif"
+  },
+  {
+     "item_id": "14",
+     "name": "Vi får väl trösta varandra",
+     "picture":"varandra.jpg"
+  },
+  {
+     "item_id": "36",
+     "name": "Kungariket",
+     "picture":"kungariket.jfif"
+  }
+ ];
+ 
+   
+  
+ function showItemId(itemId) {
+
+  var itemIdP = document.getElementsByClassName("item_id"); 
+  var image = document.getElementsByClassName("image");
+  var name = document.getElementsByClassName("name");
+
+  for (var i = 0; i < books.length; i++) {
+    var book_id=  books[i].item_id.trim();
+
+    console.log("item_nbr: "+ itemId);
+    console.log("item_nbr for varje book: "+ book_id);
+    console.log("is it equals: "+ (itemId.trim() == book_id.trim()));
+    if(itemId.trim() == book_id.trim()){
+      console.log("found it ");
+
+      itemIdP.innerHTML = `item id: `+books[i].item_id;
+      image.innerHTML= `<img src="pic/books/`+books[i].picture+`>`;
```

Comment:
For image, its better to change the src element, and use an actual `image` element instead of inside a `p` element.
Change the src element with:
`image.src = 'pic/books/...'`

Reply:
I tried this one but I had problems in style so instead  I use this one now 

  var img = document.createElement("img");
    img.src = "pic/books/"+books[i].picture;
    img.width = 100;
    img.height = 200;
    book_image.appendChild(img);